docs/installation-and-operation/serialization.md

```diff
@@ -12,116 +12,364 @@ Once you really get rolling with Metabase, it's often the case that you'll have
+  An example export could include the following directories, depending on what you exported and the contents of your Metabase:
+
+  - actions
+  - collections
+    - cards
+    - dashboards
+    - timelines
+  - databases
+
+  When serializing through the API, the export directory [will be a compressed into a .tar.gz file](#you-must-compress-your-files-when-serializing-via-api-calls).
+
+- A `settings.yaml` file that includes some [Metabase-wide settings](#the-general-settings-that-metabase-exports)
 
-An example export could include the following directories, depending on what you exported:
+Database connection details are not included by default, so you but you can [configure your export](#customize-what-gets-exported) to include them.
 
-- actions
-- collections
-- databases
+### General Metabase settings that are exported
 
-In the `collections/cards` directory, you'll see that Metabase prefixes individual files with IDs to disambiguate entities that share the same name:
+Here's the list of the general settings Metabase exports in the `settings.yaml` file. For more on Metabase settings, see [Configuring Metabase](../configuring-metabase/start.md).
 
 ```
-IA96oUzmUbYfNFl0GzhRj_accounts_model.yaml
-KUEGiWvoBFEc5oGQCEnPg_converted_customers.yaml
-qzNa8ZeFgFXrrIoF2g8m4_accounts_model_detail.yaml
+humanization-strategy
+native-query-autocomplete-match-style
+site-locale
+report-timezone-short
+report-timezone-long
+application-name
+enable-xrays
+show-homepage-pin-message
+source-address-header
+enable-nested-queries
+custom-geojson-enabled
+start-of-week
+custom-geojson
+available-timezones
+unaggregated-query-row-limit
+aggregated-query-row-limit
+hide-embed-branding?
+search-typeahead-enabled
+enable-sandboxes?
+application-font
+available-locales
+landing-page
+enable-embedding
+application-colors
+application-logo-url
+application-favicon-url
+show-homepage-xrays
+show-metabot
+enable-whitelabeling?
+show-homepage-data
+site-name
+application-font-files
+loading-message
+report-timezone
+persisted-models-enabled
+enable-content-management?
+subscription-allowed-domains
+breakout-bins-num
+available-fonts
+custom-formatting
 ```
 
-## Example question
+### Customize what gets exported
 
-Questions can be found in the `cards` directory of a collection directory. Here's an example card YAML file for a native question (a question written in SQL):
+You can customize what gets exported. You can tell Metabase:
 
-```
-description: The number of plans by referral source.
+- Export specific collections
+- Do not export collections
+- Do not export Metabase settings
+- Do not export table metadata
+- Include sample field values (excluded by default)
+- Include database connection details (excluded by default)
+
+See [export parameters in CLI commands](#use-parameters-to-customize-what-metabase-exports) or [export parameters in API calls](#api-export-parameters).
+
+### Example of a serialized question
```

Comment:
Adding a new example because it showcases more stuff. Ideally I'd like us to have a repo with the maximal possible example of serialization